src/Core/src/Stream/IterableStream.php

```diff
@@ -7,15 +7,16 @@
 /**
  * Convert an iterator into a Stream.
  *
- * @internal
- *
  * @author Jérémy Derussé <jeremy@derusse.com>
+ *
+ * @internal
+ * @final
  */
-final class IterableStream implements RequestStream
+class IterableStream implements RequestStream, ReadOnceResultStream
 {
     private $content;
 
-    private function __construct(iterable $content)
+    protected function __construct(iterable $content)
```

Comment:
This can be private (and the class final)

Reply:
indeed, forgot this